<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFoldingClass</class>
 <widget class="QMainWindow" name="CFoldingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1139</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CFolding</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="MoveTool">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Remove all selected objects</string>
    </property>
    <property name="text">
     <string>M</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="VertexTool">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select Vertex</string>
    </property>
    <property name="text">
     <string>V</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="EdgeTool">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select Edges</string>
    </property>
    <property name="text">
     <string>E</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>491</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="show2dLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>541</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="show3dLayout"/>
   </widget>
   <widget class="QToolButton" name="SymmetryTool">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="opVersaTool">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>O</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="faceTool">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Select Faces</string>
    </property>
    <property name="text">
     <string>F</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>1091</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Vertex</string>
     </attribute>
     <widget class="QScrollArea" name="candiLayout">
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>0</y>
        <width>1091</width>
        <height>181</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1089</width>
         <height>162</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Face</string>
     </attribute>
     <widget class="QScrollArea" name="faceCandi">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1081</width>
        <height>181</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1079</width>
         <height>162</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_2"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionInitialize"/>
   <addaction name="actionOptimize"/>
   <addaction name="actionLoadtest"/>
   <addaction name="actionAnimation"/>
   <addaction name="actionUndo"/>
   <addaction name="actionSave2D"/>
  </widget>
  <action name="actionInitialize">
   <property name="text">
    <string>Initialize</string>
   </property>
  </action>
  <action name="actionLoad_2">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOptimize">
   <property name="text">
    <string>Optimize</string>
   </property>
  </action>
  <action name="actionLoadtest">
   <property name="text">
    <string>loadtest</string>
   </property>
  </action>
  <action name="actionAnimation">
   <property name="text">
    <string>Animation</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionSave2D">
   <property name="text">
    <string>Save2D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CFolding.qrc"/>
 </resources>
 <connections/>
</ui>
